spec/factories/chargeback_rate.rb

```diff
@@ -15,6 +16,20 @@
       end
     end
 
+    trait :with_custom_compute_details do
+      after(:create) do |chargeback_rate, evaluator|
+        evaluator.detail_params.each do |factory_name, chargeback_rate_params|
```

Comment:
@lpichler Does it make sense to have this wrapped in a trait, even? (`with_custom_compute_details`)?

You could just wrap this in an `if evaluator.detail_params` so the signature is just `FactoryGirl.create(:chargeback_rate, :detail_params =>` (and `detail_params` is optional). Thoughts?